Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.9-alpine AS build
```

Comment:
Any particular reason to use 1.9-alpine rather than 1.10-alpine?

Reply:
Updated to 1-10.